packages/es-dev-server/src/config.ts

```diff
@@ -0,0 +1,282 @@
+import path from 'path';
+import { Middleware } from 'koa';
+import { defaultFileExtensions } from '@open-wc/building-utils';
+import { Options as NodeResolveOptions } from '@rollup/plugin-node-resolve';
+import { PolyfillsLoaderConfig } from './utils/inject-polyfills-loader';
+import { toBrowserPath, setDebug } from './utils/utils';
+import { compatibilityModes } from './constants';
+import { ResponseTransformer } from './middleware/response-transform';
+import { TransformOptions, Node } from '@babel/core';
+
+// Config object for Koa compress middleware
```

Comment:
Recommend `/** */` docblock comments for all of these, so they show up in IDE

Reply:
Nice! Thx